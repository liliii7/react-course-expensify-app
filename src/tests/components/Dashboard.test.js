import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/Dashboard";

test('should render dashboard', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot(); 
})