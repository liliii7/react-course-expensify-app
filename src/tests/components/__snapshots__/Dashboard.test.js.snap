// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFitlers) />
  <Connect(ExpenseList) />
</div>
`;
